agot-bg-game-server/src/client/IngameComponent.tsx

```diff
@@ -569,26 +574,44 @@ export default class IngameComponent extends Component<IngameComponentProps> {
 
     private renderRemainingWesterosCards(): ReactNode {
         const remainingCards = this.game.remainingWesterosCardTypes;
-
+        const nextCards = this.game.nextWesterosCardNames;
```

Comment:
Can you please apply the same logic as for Remaining Westeros Cards and save the card type instead only the name. That will allow improving the ToolTip some day and maybe also show the description or even the images. Users already requested if that would be possible as not everyone knows all cards by heart.

Reply:
changed the function name to nextWesterosCardTypes, now it returns WesterosCardType instead of string